### ISSUE TITLE
VoiceOver: Overpayment alerts aren't announcing as collapsed

### ISSUE THREAD

**Issue by nhuckleberry6** [department-of-veterans-affairs/va-mobile-feature-support#1228]
# Description

Alerts within Overpayments aren't announced as collapsed by VoiceOver. However, they are announcing correctly in Copays. 

1. Log in to demo mode with the overpayments and copays feature flags and VoiceOver on
2. Double-tap the overpayment tile on Home
3. Double-tap on review details for the first debt
4. Swipe right until you get to the alert heading; it doesn't announce "collapsed"
6. Navigate to copays
7. Double-tap on review details for the first copay
8. Swipe right until you get to the alert heading which announces "collapsed"

# Additional Info and Resources

- [Jon posted a video in Slack 🧵 of Copays working correctly](https://dsva.slack.com/archives/C018V2JCWRJ/p1773328939294089?thread_ts=1772721691.471469&cid=C018V2JCWRJ)
- This is an issue with other alerts in the app as well. Jon noted, "I'd expect it to be setup specific since it's so hit-or-miss, and probably not in a way that anyone would be able to know intuitively."
- Eli and Jon may be meeting about guidance for this.

# Test Users

[vets.gov.user+90@gmail.com](mailto:vets.gov.user+90@gmail.com) - doesn't have copays

# Acceptance Criteria
- [ ] All collapsed alerts within overpayments are announced as "collapsed" by a screenreader.
